.github/workflows/stepci-2.8.2.yml

```
@@ -0,0 +1,27 @@
+name: stepci/stepci@2.8.2
+on:
+  pull_request:
+    paths:
+      - .github/workflows/reusable-docker-workflow.yml
+      - .github/workflows/stepci-2.8.2.yml
```

Please follow the naming convention of the workflows for this file, so:


```suggestion
      - .github/workflows/stepci-stepci-v2.yml
```

Don't forget to update the filename, other path filter, and `README.md` accordingly.